meteo-notebook.ipynb

```diff
@@ -110,7 +110,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    plt.ylabel('pm2.5')
</pre><p>Use this instead:</p><p>'Inhalable particles with diameter \n of 2.5 micrometers or smaller'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>